src/components/Books.js

```diff
@@ -3,17 +3,39 @@ import PropTypes from 'prop-types';
 
 const Books = ({ author, title, handleClick }) => (
   <div className="book">
+    <div className="bookDetails">
+      <div>
+        <small className="bookCategory">Action</small>
+        <h2 className="bookTitle">{title}</h2>
+        <p className="bookAuthor">{author}</p>
+        <div className="actionBtn">
+          <button type="button">Comments</button>
+          <button type="button" onClick={handleClick}>Remove</button>
+          <button type="button">Edit</button>
+        </div>
+      </div>
+      <div className="readingProgress">
+        <div className="clircleWrap">
+          <div className="circle" />
+        </div>
+        <div>
+          <p className="percentage">64%</p>
+          <p className="progressCompleted">Completed</p>
+        </div>
+      </div>
+    </div>
+
     <div>
-      <p>{author}</p>
-      <h2>{title}</h2>
+      <p className="progressCompleted">CURRENT CHAPTER</p>
+      <p className="currentChapter">Chapter 17</p>
+      <button
+        type="button"
+        className="btn_blue"
+      >
+        UPDATE PROGESS
+      </button>
     </div>
-    <button
-      type="button"
-      className="btn_blue"
-      onClick={handleClick}
-    >
-      Remove
-    </button>
```

Comment:
- [ ] Kindly add the border between the  buttons in the `actionBtn` and between the reading progress div and chapters, this will make your design look EXACTLY like the figma design, see screenshots below good job so far 👍🏿 💪🏿 

Zephin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/69638013/172427829-456cc1f9-9a11-4726-a890-55b5e484bee4.png) | ![Screenshot_236](https://user-images.githubusercontent.com/69638013/172427949-119106da-d97d-40b1-ae09-f356f16589c7.png)